legacy/src/app/partials/directives/release-options.html

```diff
@@ -1,5 +1,5 @@
 <div ng-if="loading">
-  <i class="p-icon--spinner u-animation--spin" />
+  <i class="p-icon--spinner u-animation--spin"></i>
   &nbsp;Loading...
```

Comment:
grepping the code, this was the only remaining instance of a self closing tag I could find, other than `input`s, `hr`s and `br`s, but perhaps have a quick grep when QAing this to ensure I haven't missed anything.